src/gwt/src/org/rstudio/studio/client/workbench/addins/Addins.java

```diff
@@ -64,7 +74,8 @@ public final static RAddin decode(String decoded)
                splat[1],
                splat[2],
                splat[3],
-               splat[4]);
+               splat[4] == "false" ? false : true,
```

Comment:
Maybe compare to "true" for consistency with how this is handled on the server side?